modules/platforms/cpp/ignite/client/ignite_client_configuration.h

```diff
@@ -77,6 +85,26 @@ class ignite_client_configuration {
         ignite_client_configuration::m_endpoints.assign(endpoints.begin(), endpoints.end());
     }
 
+    /**
+     * Set endpoints.
+     *
+     * Examples of supported formats:
+     * - 192.168.1.25 - Default port is used, see DEFAULT_PORT;
+     * - 192.168.1.25:780 - Custom port;
+     * - 192.168.1.25:780..787  - Custom port range - ports are checked from
```

Comment:
We agreed to remove port ranges (see https://issues.apache.org/jira/browse/IGNITE-19601) - should we get rid of them right here?

Reply:
I'll do that for ODBC, but I'd prefer to do it in a separate ticket for client. I remove port ranges from the comments though.